packages/validator/src/spec/PwaValidatorSpec.ts

```diff
@@ -99,5 +114,193 @@ describe('PwaValidator', () => {
           .toBe('https://developers.google.com/speed/pagespeed/insights/' +
               '?url=https%3A%2F%2Fexample.com%2F');
     });
+
+    // FCP Tests
+    it('FCP is PASS when 0', async () => {
+      const psiResult = mockPsiResult(0.8, 1.0, {
+        firstContentfulPaint: 0,
+        largestContentfulPaint: 0,
+        maxPotentialFID: 0,
+        cumulativeLayoutShift: 0,
+      });
+      const pwaValidator = mockPwaValidator(psiResult);
+      const result = await pwaValidator.validate(new URL('https://example.com'));
+      expect(result.scores.firstContentfulPaint.status).toBe('PASS');
+      expect(result.scores.firstContentfulPaint.printValue).toBe('0.0 s');
+    });
+    it('FCP is PASS when 10s', async () => {
+      const psiResult = mockPsiResult(0.8, 1.0, {
+        firstContentfulPaint: 10000,
+        largestContentfulPaint: 0,
+        maxPotentialFID: 0,
+        cumulativeLayoutShift: 0,
+      });
+      const pwaValidator = mockPwaValidator(psiResult);
+      const result = await pwaValidator.validate(new URL('https://example.com'));
+      expect(result.scores.firstContentfulPaint.status).toBe('PASS');
+      expect(result.scores.firstContentfulPaint.printValue).toBe('10.0 s');
+    });
```

Comment:
Don't we want a test for FCP is `WARN` or `FAIL` to complete the set?

Reply:
FCP isn't actually on https://web.dev/vitals/, so we don't have thresholds defined. I'm wondering it's better not to show it at all (not sure why I added in the first place)

Reply:
Oh right. Yeah, I'd vote for getting rid of it just for code (and output) simplicity.